backend/src/main/java/org/eclipse/tractusx/puris/backend/common/edc/logic/service/EdcAdapterService.java

```diff
@@ -151,30 +172,32 @@ public boolean registerAssetsInitially() {
      */
     public boolean createPolicyAndContractDefForPartner(Partner partner) {
         boolean result = createPolicyDefinitionForPartner(partner);
-        result &= createItemStockSubmodelContractDefinitionForPartner(partner);
+        result &= createSubmodelContractDefinitionForPartner("ItemStock", variablesService.getItemStockSubmodelApiAssetId(), partner);
```

Comment:
As the other parameters use a black, please also use it for Stock. 

Should we maybe just log the semanticId and use the SubmodelType? Actually, I don't have a preference here

Reply:
Using the asset id actually makes sense. There's no need to add an extra parameter just for logging purposes.